### PR TITLE
[desktop] Release notes for 2022.07.27-3

### PR DESCRIPTION
**Pull request by Ferenc-**
Signed-off-by: Ferenc Géczi <ferenc.gm@gmail.com>

### CODE REVIEW

File: qt/res/app.organicmaps.desktop.metainfo.xml [hunk 1]
```diff
@@ -107,6 +107,22 @@
...
+         <li>Fixed invalid opertions on Wayland for OpenGL ES3 capable devices</li>
+         <li>Use OpenGL ES3 by default on Linux</li>
+         <li>New OSM data as of July 18</li>
+         <li>Fixed/updated API (see omaps.app/api)</li>
+         <li>Added missing shop types from top OSM tags</li>
+         <li>Minor routing improvement</li>
+         <li>Use min_level for 3D buildings</li>
+         <li>Allow removing cuisines from selection in the Editor</li>
+         <li>Fixed be, de, ru, uk translations</li>
+         <li>Fixed FB URLs validation</li>
```
Comment (biodranik) [inline qt/res/app.organicmaps.desktop.metainfo.xml:122]: The last 3 entries are not relevant to the desktop version.

See https://github.com/flathub/app.organicmaps.desktop/pull/13#pullrequestreview-1054857409

**Review by biodranik — CHANGES_REQUESTED** [review]

**Comment by fgaz** [comment]
I think we could avoid such problems and delays in the future by having the changelog already present upstream at time of release. That's why I was working on a CI check for that in #2866. If that's a no-go I think we could just omit the changelog entirely and skip this whole process.

**Comment by biodranik** [comment]
Can we make a CI check in a such way, that if the changelog file will be updated, then it will trigger the release automatically?
I would also prefer to have a simple text file for the desktop changelog, probably in the repo's root. Like CHANGELOG.md or CHANGELOG.txt
It would be also great to store there the full changelog for all platforms like it's done here: https://t.me/s/OrganicMapsApp

I also like the idea of keeping the previous history. The main pain point now is manual editing and formatting of files in different places...

**Comment by fgaz** [comment]
> Can we make a CI check in a such way, that if the changelog file will be updated, then it will trigger the release automatically?

looks like it's possible: https://github.com/flathub/flatpak-external-data-checker

For the changelog, maybe there could be a small script to convert it from md to xml?

File: qt/res/app.organicmaps.desktop.metainfo.xml (hunk 1 above)
Comment (Ferenc-) [reply qt/res/app.organicmaps.desktop.metainfo.xml]: OK the last 3 entries have been removed.

**Review by biodranik — APPROVED** [review]

**Pull request merged**